spec/helpers/administrate/application_helper_spec.rb

```diff
@@ -41,4 +41,43 @@
       end
     end
   end
+
+  describe "#svg_tag" do
+    subject { Nokogiri::HTML svg_tag('logo.svg',
+                                    svg_id: 'logo')}
+
+    it "returns use tag with xlink:href attribute" do
+      use_tag = subject.xpath("//svg//use['xlink:href']")
+
+      expect(use_tag.length).to eq(1)
+    end
+
+    it "returns use tag with svg file path in xlink:href attribute" do
+      use_tag = subject.xpath("//svg//use['xlink:href']")
+      attribute = use_tag[0].attributes["xlink:href"]
+
+      expect(attribute.value).to eq("/logo.svg#logo")
+    end
+
+    context "with size attributes" do
+      subject { Nokogiri::HTML svg_tag('logo.svg',
+                                      svg_id: 'logo',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.